Makefile

```diff
@@ -27,16 +33,23 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	pip install -qr requirements/pip-tools.txt
 	$(PIP_COMPILE) -o requirements/pip-tools.txt requirements/pip-tools.in
 	$(PIP_COMPILE) -o requirements/base.txt requirements/base.in
+	$(PIP_COMPILE) -o requirements/quality.txt requirements/quality.in
 	$(PIP_COMPILE) -o requirements/test.txt requirements/test.in
+	$(PIP_COMPILE) -o requirements/tox.txt requirements/tox.in
+
+python-quality-test: clean ## check coding style with pycodestyle and pylint
+	$(TOX) pycodestyle --config=setup.cfg ./openedx_external_enrollments
```

Comment:
MIssing quality requirements installation step. @ivanvgh

Reply:
The 'run-tests' command installs the dependencies through 'install-dev-dependencies' command.